docs/walkthrough/integrate-with-jina.md

```diff
@@ -2,6 +2,7 @@
 
 Once fine-tuning is finished, it's time to actually use the model.
 You can use the fine-tuned models directly to encode [DocumentArray](https://docarray.jina.ai/) objects or setting up an encoding service.
+It is worth noting that, while training data can be provided as a csv, data for encoding cannot.
```

Comment:
Why not?

Reply:
Data would be structured differently in the csv, meaning a new method would need to be written, this could be added if it important.

Reply:
I think we should be consistent everywhere we accept datasets. If we want to make CSV the primary format this should apply for all the API in total. Generally our build_dataset function should support unlabeled data as well. @bwanglzu 

If we leave this for another PR we are creating technical debt

Reply:
yes we should not leave technical debts, everywhere should be consistent